parts/kubernetesmastercustomdata.yml

```diff
@@ -1,7 +1,9 @@
 #cloud-config
 
 packages:
+{{if not .OrchestratorProfile.IsCustomEtcdVersion}}
```

Comment:
I would almost argue that we don't use the upstream packages and remove this entirely. The upstream apt packages don't even use the last 2 dot release which contains a bunch of operational fixes. Worth it or do we let etcd2 die on the vine as we will soon deprecate k8s 1.5

Reply:
I agree. But should happen outside this PR. This PR is focused on etcd3. The next should focused on using the same binary source for all releases.